src/settings/ReadingRoomAccess/getValidators.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/prefer-default-export */
+import { FormattedMessage } from 'react-intl';
+import { readingRoomAccessColumns } from './constant';
+
+const validators = {
+  [readingRoomAccessColumns.NAME]: function validateName(item) {
```

Comment:
Is there is any reason use name function instead of just arrow? Looks like it calls by key and this name is visible only here.

Reply:
updated to anonymous arrow functions